tests/new_api/test_device.py

```diff
@@ -1176,32 +1177,25 @@ def test_vjp_tape_batch(self, phi, execute_and_derivatives, batch_obs):
         qs2 = QuantumScript(ops, [qml.expval(qml.prod(qml.Z(0), qml.Z(1)))], trainable_params=[0])
         dy = [(1.5, 2.5), 1.0]
 
+        expected_device = DefaultQubit(wires=4, max_workers=1)
+
         if execute_and_derivatives:
             results, jacs = device.execute_and_compute_vjp((qs1, qs2), dy)
+
+            expected, expected_jac = expected_device.execute_and_compute_vjp((qs1, qs2), dy)
         else:
             results = device.execute((qs1, qs2))
             jacs = device.compute_vjp((qs1, qs2), dy)
 
-        # Assert results
-        expected1 = (-np.sin(phi) - 1, 3 * np.cos(phi))
-        x1 = np.cos(phi / 2) ** 2 / 2
-        x2 = np.sin(phi / 2) ** 2 / 2
-        expected2 = sum([x1, -x2, -x1, x2])  # zero
-        expected = (expected1, expected2)
```

Comment:
Analytic results have some advantages, like DQ and LQ could be consistently wrong. On the other hand DQ has tons of tests using analytic results, and hence can usually be used as a source of truth. Just pointing it out.